modules/tinymce/src/core/main/ts/view/VisualAidsImpl.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Arr } from '@ephox/katamari';
+import Editor from '../api/Editor';
+import * as Settings from '../api/Settings';
+
+export const addVisualInternal = (editor: Editor, elm?: HTMLElement) => {
+  const dom = editor.dom;
+  elm = elm || editor.getBody();
```

Comment:
Should we make this a new variable/const? Not sure how much you want to change here, given this is largely just moved code.